front-end/src/renderer/components/ui/AppCheckBox.vue

```diff
@@ -6,6 +6,7 @@ defineProps<{
   label?: string;
   disabled?: boolean;
   dataTestid?: string;
+  type?: 'checkbox' | 'radio';
```

Comment:
The AppCheckBox component is a separate component from the radio button.
Please remove the type property